index.html

```diff
@@ -2518,14 +2518,16 @@ <h3>
           When an <a>ImageResource</a> is used as an <dfn>icon</dfn>, a
           developer can hint that the image is intended to serve some special
           purpose in the context of the host OS (i.e., for better integration).
+          User agents SHOULD NOT use an icon other than for its stated purpose.
```

Comment:
Do we have a concept of "purpose" defined? 

```suggestion
          User agents SHOULD NOT use an icon other than for its stated <a>purpose</a>.
```